test/Transforms/pipeline-data-transfer.mlir

```diff
@@ -52,7 +52,7 @@ func @loop_nest_dma() {
 // CHECK-NEXT:  %{{.*}} = affine.load %{{.*}}[%{{.*}} mod 2, %{{.*}}] : memref<2x32xf32, 1>
 // CHECK-NEXT:  %{{.*}} = "compute"(%{{.*}}) : (f32) -> f32
 // CHECK-NEXT:  affine.store %{{.*}}, %{{.*}}[%{{.*}} mod 2, %{{.*}}] : memref<2x32xf32, 1>
-// CHECK-NEXT:  affine.for %{{.*}} = 0 to 128 {
+// CHECK-NEXT:  affine.for %{{.*}} = 0 to 32 {
```

Comment:
Why these changes 128->32?

Reply:
This one is minor and orthogonal. The DMA buffer size / transfer size in the op's above is actually 32, and so the loop should be of trip count 32 to be meaningful. It doesn't affect this test case - but just fixed it while on this file.

Reply:
Sure. (I was under the impression that cleanup/NFC in the same file was fine.)